### PR TITLE
Add Rejoining the Beatmap Nominators requirements

### PR DESCRIPTION
**Pull request by Okorin**
Adds the requirements for rejoining the BN after leaving for whichever reason to the becoming a BN article

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md [hunk 1]
```diff
@@ -54,3 +54,22 @@ The following are modding traits expected of Beatmap Nominators. Failure to disp
...
+## Rejoining the Beatmap Nominators
+
+Users who have recently left the Beatmap Nominators may have different requirements to rejoin compared to when they first joined. For users who have performed well prior to leaving, they may be able to rejoin with relaxed requirements. The NAT will inform the user if they have the opportunity to fast-track their way back into the Beatmap Nominators.
+
+### Standard Rejoining Requirements
+
+Rejoining the Beatmap Nominators with standard requirements is treated similarly to failing an application.
+- The user must reapply to the Beatmap Nominators via the [website](http://bn.mappersguild.com/bnapps), with a `Mod Score` of at least 0 over a period of 90 days.
+- There is a 90 day cooldown period starting from the user's removal from the Beatmap Nominators before they are able to reapply.
+- The user must provide 2 to4 recent mods in the application and retake the [Ranking Criteria](/wiki/Ranking_Criteria) test as normal.
```
Comment (zeusminus) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:68]:
```suggestion
- The user must provide 2 to 4 recent mods in the application and retake the [Beatmap Nominator Test](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Nominator_Test) as normal.
```
might as well toss the relevant article here

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md [hunk 2]
```diff
@@ -54,3 +54,22 @@ The following are modding traits expected of Beatmap Nominators. Failure to disp
   - General attitude: Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Ranking_Criteria/Code_of_Conduct), as well as general behaviour in chat/forums.
   - Modding abilities: Knowledge of the general [Ranking Criteria](/wiki/Ranking_Criteria) and specific criteria of each game mode. Additional abilities like Metadata, Timing, and some others will also be taken into consideration.
 - After evaluating each modder, a discussion will be made on whether the modder will join the Beatmap Nominators.
+
+## Rejoining the Beatmap Nominators
+
+Users who have recently left the Beatmap Nominators may have different requirements to rejoin compared to when they first joined. For users who have performed well prior to leaving, they may be able to rejoin with relaxed requirements. The NAT will inform the user if they have the opportunity to fast-track their way back into the Beatmap Nominators.
+
+### Standard Rejoining Requirements
```
Comment (zeusminus) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:62]:
```suggestion
### Standard rejoining requirements
```
sentence-case is mandatory

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md [hunk 3]
```diff
@@ -54,3 +54,22 @@ The following are modding traits expected of Beatmap Nominators. Failure to disp
...
+
+Rejoining the Beatmap Nominators with standard requirements is treated similarly to failing an application.
+- The user must reapply to the Beatmap Nominators via the [website](http://bn.mappersguild.com/bnapps), with a `Mod Score` of at least 0 over a period of 90 days.
+- There is a 90 day cooldown period starting from the user's removal from the Beatmap Nominators before they are able to reapply.
+- The user must provide 2 to4 recent mods in the application and retake the [Ranking Criteria](/wiki/Ranking_Criteria) test as normal.
+
+### Fast-track Rejoining Requirements
```
Comment (zeusminus) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:70]:
```suggestion
### Fast-track rejoining requirements
```
sentence-case is mandatory

**Review by zeusminus — CHANGES_REQUESTED** [review]

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ The following are modding traits expected of Beatmap Nominators. Failure to disp
 `m` = Minimum expected number of individual beatmaps modded in a month\
 `M` = Total achieved number of individual beatmaps modded in a month
 
-- [The website](http://bn.mappersguild.com/bnapps) has a tool for calculating mod score on its main page.
+- [The website](http://bn.mappersguild.com/bnapps) has a tool for calculating `Mod Score` on its main page.
```
Comment (peppy) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:43]: Is this how we use backticks in the wiki? Generally it's only used for variable names.. so something like bold or quotations may work better for terms like this?

Comment (Okorin) [reply wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:43]: The rest of the article formatted it this way, I personally have no idea or opinion on what is correct here

Comment (peppy) [reply wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:43]: Would need to check the wiki style guide - should be fixed to match what we do everywhere else.

Comment (Okorin) [reply wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:43]: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#keywords-and-commands could apply? I don't know - it is used as a keyword

**Review by TicClick — APPROVED** [review]
a formal approval from me w.r.t formatting/contents/meaning, but it's not my call to make

**Review by Joehuu — DISMISSED** [review]
LGTM

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md [hunk 5]
```diff
@@ -33,24 +33,44 @@ The following are modding traits expected of Beatmap Nominators. Failure to disp
...
+
+## Rejoining the Beatmap Nominators
+
+Users who have recently left the Beatmap Nominators may have different requirements to rejoin compared to when they first joined. For users who have performed well prior to leaving, they may be able to rejoin with relaxed requirements. The NAT will inform the user if they have the opportunity to fast-track their way back into the Beatmap Nominators.
+
+### Standard rejoining requirements
+
+Rejoining the Beatmap Nominators with standard requirements is treated similarly to failing an application.
+
+- The user must reapply to the Beatmap Nominators via the [website](http://bn.mappersguild.com/bnapps), with a `ModScore` of at least 0 over a period of 90 days.
```
Comment (pishifat) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md:66]: currently returning bns who aren't "fast-tracked" need modscore>0 for 60 days. is the 90 here intentional?

Comment (Okorin) [reply wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md]: not to my knowledge

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**